corehq/apps/reports/v2/filters/case_columns.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy
+
+from corehq.apps.es import queries
+from corehq.apps.reports.v2.models import (
+    BaseColumnFilter,
+    FilterChoiceMeta,
+)
+
+
+class TextCaseColumnFilter(BaseColumnFilter):
```

Comment:
It would be great to have tests for all of this

Reply:
will address in another PR